R/o2.at.sat.R

```diff
@@ -0,0 +1,6 @@
+#' Calculates the equilibrium saturation concentration of oxygen in water at the supplied conditions
+#' 
+#' @importFrom LakeMetabolizer o2.at.sat
+o2.at.sat <- function(ts.data, baro, altitude=0, salinity=0, model='garcia'){
+  LakeMetabolizer::o2.at.sat(ts.data, baro, altitude, salinity, model)
```

Comment:
do we want to pull in o2.at.sat.base instead? Same question applies to the other ones that have a `ts.data` argument